autoprotocol/util.py

```diff
@@ -226,6 +227,36 @@ def check_stamp_append(current_xfer, prev_xfer_list, maxTransfers=3,
     return True
 
 
+def check_valid_mag(container, head):
+    if head == "96-deep":
+        if container.container_type.shortname not in ["96-v-kf", "96-deep-kf", "96-deep", "96-flat", "96-flat-uv"]:
```

Comment:
could probably either make these container type lists constants or do the check based on well_count && well_volume_ul